preprocessing_env.yml

```diff
@@ -0,0 +1,22 @@
+name: alsf_preprocessing_env
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::pip
+  - conda-forge::tifffile
+  - conda-forge::jupyterlab
+  - conda-forge::pandas=1.4.4
+  - conda-forge::ipykernel
+  - conda-forge::nb_conda_kernels
+  - conda-forge::scipy=1.10.0
+  - conda-forge::numpy=1.22
+  - conda-forge::scikit-learn
+  - conda-forge::seaborn
+  - conda-forge::umap-learn
+  - conda-forge::matplotlib
+  - pip:
+    - pycytominer
```

Comment:
I recommend moving over to conda for pycytominer install and specifying version: https://anaconda.org/conda-forge/pycytominer/files

Reply:
Sounds good! I will add it now!